### PR TITLE
remember_me

### PR DESCRIPTION
**Pull request by AilinSoon**
ready

### CODE REVIEW

**Comment by AilinSoon** [comment]
ready

File: app/views/shared/_error_messages.html.erb [hunk 1]
```diff
@@ -1,13 +1,13 @@
-<% if @user.errors.any? %>
-  <div id="error_explanation">
-    <div class="alert alert-danger">
-      <%= t (:the_form_contains)%>
-      <%= pluralize @user.errors.count, t(:error) %>.
-    </div>
-    <ul>
-      <% @user.errors.full_messages.each do |msg| %>
+<% if object.errors.any? %>
+    <div id="error_explanation">
```
Comment (huongnt-2545) [inline app/views/shared/_error_messages.html.erb]: wrong indent

File: config/routes.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@
       post "/signup", to: "users#create"
       get "/login", to: "sessions#new"
       post "/login", to: "sessions#create"
-      delete "/logout", to: "sessions#destroy"
```
Comment (huongnt-2545) [inline config/routes.rb]: keep this, don't change

**Comment by AilinSoon** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**